File: all/cap01/casos/EP1_2.java
import java.util.Scanner;
class EP1_2 {
  public static void main(String[] args) {
    Scanner teclado1 = new Scanner(System.in);
    int C = teclado1.nextInt();
    float F = C * 9/5 + 32;
    System.out.println(C + " graus Celsius corresponde a " + F + " graus Fahrenheit");
  }
}

File: all/cap01/casos/EP1_2.js
function escreva(s) { try { document.write(s); } catch(e) { console.log(s); } }
function ler(linhas, i) { return linhas.toString().split('\n')[i]; }

process.stdin.once('data', linhas => {
  var C = ler(linhas, 0); // ATENÇÃO NESTA INSTRUÇÃO!
  var F = C * 9/5 + 32;
  escreva(C + " graus Celsius corresponde a " + F.toFixed(1) + " graus Fahrenheit");
});